### DISCUSSION TITLE
Orientation colouring

### DISCUSSION THREAD

**Discussion by sushdash** [neperfepx/neper#444] [Q&A] [answered]
Hello!
I have uniaxially deformed a 3D sample along Y direction up to 2% strain. After this, I want to visualize this using the "ipf" coloring scheme, for which I am using

neper -V ../1_uniaxial.sim \
    -simstep 2 \
    -datanodecoo coo \
    -datanodecoofact 1 \
    -dataelt3dcol ori:internal \
    -dataelt3dcolscheme ipf \
    -cameraangle 14.5 \
    -showelt1d none \
    -imagesize 800:800 \
    -print 2_ori

However, by default its taking "ipf along z". But I want it to be along "y". 

Please help me with this.

**Comment by rquey** [comment, marked as answer]
Use [-dataelt3dcolscheme ipf(y)](https://neper.info/doc/neper_v.html#cmdoption-data-elt-elset-node-elt-0-3-d-elset-0-3-d-colscheme).
